### DISCUSSION TITLE
multiselect component

### DISCUSSION THREAD

**Discussion by DanielFGray** [tailwindlabs/headlessui#1201] [Ideas]
It would be fantastic to have a multiselect dropdown component, similar in functionality to <https://mantine.dev/core/multi-select/>

**Comment by vhscom** [comment]
What would differentiate a _Multiselect_ dropdown component from a _Listbox_ with multi-select as proposed in https://github.com/tailwindlabs/headlessui/discussions/668?

**Reply by DanielFGray** [reply]
A listbox typically presents a finite list of selections

The multiselect I have in my mind allows you to narrow the selection by typing and should also [optionally] allow creating your own input (think stackoverflow tags)

**Reply by vhscom** [reply]
The key differentiator I see in the Mantine multi-select docs are the chips inside the input as [one user hinted at](https://github.com/tailwindlabs/headlessui/discussions/668#discussioncomment-963363) in #668 (now in quasi beta). Some selection narrowing is available inside the already released [Combox](https://headlessui.dev/react/combobox) component. When I think about tagging on SO I think about a spotlight search like: https://svelte-spotlight.vercel.app. I don't see a spotlight search in Mantine.
